### DISCUSSION TITLE
Beginner question

### DISCUSSION THREAD

**Discussion by al-davis** [schreibfaul1/ESP32-audioI2S#714] [Q&A]
Day one running I2sAudio example, came up very easy.  My goal to play live streams from a variety of websites that I listen to via my 
desktop.    Is that possible ?    So far no luck, for instance I tried  adding this line of code :
audio.connecttohost("https://d1qg6pckcqcdk0.cloudfront.net/classical2/parkeningchrist_christopherpark_17_variationsonathemefr.m4a");
   but the result was a short burst of gibberish.  Did I pick a format that is not compatible ?   Am I on the wrong track ?

**Comment by schreibfaul1** [comment]
You have caught a stream that is not currently running. The file is causing some kind of problem, I have to analyse it first.
